newton/_src/solvers/kamino/_src/linalg/conjugate.py

```
@@ -151,132 +180,153 @@ def matvec(self, x: wp.array2d, y: wp.array2d, world_active: wp.array):
+    world_active: wp.array[wp.int32],
+    cur_iter: wp.array[int],
+    world_condition: wp.array[wp.int32],
+    batch_condition: wp.array[wp.int32],
+):
+    wid = wp.tid()
+
+    # Update iteration
+    active = world_active[wid]
+    condition = world_condition[wid]
+    world_stepped = active * condition
+    iter = world_stepped * loop_granularity + cur_iter[wid]
```

Ruff (0.15.12)

[error] Variable `iter` is shadowing a Python builtin

(A001)